styles.css

```diff
@@ -3,3 +3,46 @@
 *::after {
   box-sizing: border-box;
 }
+* {
+  font-family: "Maven Pro", sans-serif;
+}
+body {
+  background-color: #141414;
+  text-decoration: none;
+  margin: 0;
+}
+a {
+  text-decoration: none;
+}
+header {
+  height: 100px;
```

Comment:
Instead of relying on a fixed `height` here, I'd recommend having a grid in the parent element. That way, you can get away from the fixed size for your element, which makes it very unresponsive.